api/notd/api/endpoints_v1.py

```diff
@@ -169,3 +170,15 @@ class UpdateAttributeForAllTokensDeferredRequest(BaseModel):
 
 class UpdateAttributeForAllTokensDeferredResponse(BaseModel):
     pass
+
+class GetCollectionAttributesResponse(BaseModel):
+    attributes: List[ApiAttribute]
+
+class GetCollectionAttributesRequest(BaseModel):
+    pass
+
+class GetTokensWithAttributeResponse(BaseModel):
```

Comment:
This should just be GetCollectionTokens. it should be the same everywhere (function names etc). We will filter by more things in the future, not just attributes